build_docs.sh

```diff
@@ -28,29 +28,27 @@ doc_files=(
 	'Blackbox.md'
 	'Migrating from baseflight.md'
 	'Boards.md'
-	'Board - AlienFlight.md'
-	'Board - CC3D.md'
-	'Board - CJMCU.md'
-	'Board - Naze32.md'
-	'Board - Sparky.md'
-	'Board - Olimexino.md'
-	'Board - ChebuzzF3.md'
+	'./boards/Board - AlienFlight.md'
+	'./boards/Board - CC3D.md'
+	'./boards/Board - CJMCU.md'
+	'./boards/Board - Naze32.md'
+	'./boards/Board - Sparky.md'
+	'./boards/Board - Olimexino.md'
+	'./boards/Board - ChebuzzF3.md'
 )
 
 if which gimli >/dev/null; then
-	echo "Building ${filename}.pdf"
+	echo "Building ${filename}"
 	pushd . >/dev/null
 	cd docs
-
-	rm -f ${filename}.md
+	rm -f $filename
 	for i in "${doc_files[@]}"
 	do
-		cat "$i" >> ${filename}.md
+		cat "$i" >> $filename
+
 	done
-	rm -f ${filename}.pdf
-	gimli -f ${filename}.md -stylesheet override.css \
-	  -w '--toc --title "Cleanflight Manual" --footer-right "[page]" --toc-depth 1'
-	rm ${filename}.md
+        gimli -f {"$filename"}.md -stylesheet override.css \
```

Comment:
This won't work. You are first creating a file called `Manual` above, and now you are telling gimli to process a file called `{Manual}.md`, which obviously does not exist.

One shortcoming of gimli is that from the looks of it, it just terminates without an error message if the specified input file does not exist. Do not confuse this with 'it's working'.

Also, I told you before that the input that you are trying to get processed by gimli (essentially a concatenation of all of the .md files under `docs/`) has got markdown syntax errors (three backticks not being on a line of their own being one of them), and they will have to be fixed before this will work again.

Reply:
Ok, that makes more sense. Gimly would crash if I set the .md extension explicitly. Without it, it would generate a markdown file called "Manual" If you change the filename variable to "Manual.md", the file generated would be Manual.md.

When I get out of class, I'll play around with the markdown of the files.